### PR TITLE
add selectors for datavolume annotations

### PR DESCRIPTION
**Pull request by yaacov**
Add selectors specific for datavolume annotations from common templates

Part of: #5977

### CODE REVIEW

File: frontend/packages/kubevirt-plugin/src/k8s/wrapper/vm/data-volume-wrapper.ts [hunk 1]
```diff
@@ -106,6 +106,18 @@ export class DataVolumeWrapper extends K8sResourceObjectWithTypePropertyWrapper<
     return this;
   };
 
+  setPesistentVolumeClaimName = (name: string) => {
```
Comment (irosenzw) [inline frontend/packages/kubevirt-plugin/src/k8s/wrapper/vm/data-volume-wrapper.ts:109]: What if `name` is null or undefined ?
Is there any need to validate the name (to check for non-allowed chars) ?

File: frontend/packages/kubevirt-plugin/src/k8s/wrapper/vm/data-volume-wrapper.ts [hunk 2]
```diff
@@ -106,6 +106,18 @@ export class DataVolumeWrapper extends K8sResourceObjectWithTypePropertyWrapper<
...
+    this.ensurePath('spec.source.pvc');
+    this.data.spec.source.pvc.name = name;
+    return this;
+  };
+
+  setPesistentVolumeClaimNamespace = (namespace: string) => {
```
Comment (irosenzw) [inline frontend/packages/kubevirt-plugin/src/k8s/wrapper/vm/data-volume-wrapper.ts:115]: same here.

File: frontend/packages/kubevirt-plugin/src/k8s/wrapper/vm/data-volume-wrapper.ts (hunk 1 above)
Comment (yaacov) [reply frontend/packages/kubevirt-plugin/src/k8s/wrapper/vm/data-volume-wrapper.ts:109]: we don't do this checks in other places ... IMHO it's ok to let the user of this method do the checks before calling this method

**Pull request merged**